src/wallet/rpcwallet.h

```diff
@@ -28,4 +28,5 @@ bool EnsureWalletIsAvailable(CWallet *, bool avoidException);
 
 UniValue getaddressinfo(const JSONRPCRequest& request);
 UniValue signrawtransactionwithwallet(const JSONRPCRequest& request);
+bool FillPSBT(const CWallet* pwallet, PartiallySignedTransaction& psbtx, const CTransaction* txConst, int sighash_type = 1, bool sign = true);
```

Comment:
why not pass by reference if it's going to be there?

Reply:
Which?

Reply:
was referring to pwallet, which you dispute as an improvement